backend/src/routes/orders/index.ts

```diff
@@ -38,6 +38,7 @@ ordersRouter.get(
     }
 
     const result = await dataSource.manager.findAndCount(Order, {
+      relations: ["invoices", "offer", "overdue_notices", "client"],
```

Comment:
This part is a bit tricky actually.

An employee (this is a user role) should not get data about the sub orders 🤔

Reply:
Could you please adjust the backend when you have time?